### PR TITLE
Cart operations GraphQl Schema actualization and in store pickup support

### PR DESCRIPTION
**Pull request by swnsma**
## Problem
1. Current GraphQl Schema for Address Operations is out-of date  with the implementation.
2. It is required to have possibility to pass 'Pickup Location Code' to the shipping address via GraphQl (see https://github.com/magento/inventory/pull/2427)

## Solution
1. Actualize schema with the latest changes from Magento Codebase (first commit: 
0d1f9ef)
2. Add support of 'Pickup Location Code' field (second commit: 
9602dbb)

## Requested Reviewers
@paliarush

### CODE REVIEW

File: design-documents/graph-ql/coverage/CartAddressOperations.graphqls [hunk 1]
```diff
@@ -55,30 +62,32 @@ type SetBillingAddressOnCartOutput {
...
-    telephone: String!
-    address_type: AdressTypeEnum!
-
-    selected_shipping_method: CheckoutShippingMethod
-    available_shipping_methods: [CheckoutShippingMethod]!
-
-    items_weight: Float
+    country: CartAddressCountry
+    telephone: String
     customer_notes: String
```
Comment (joni-jones) [inline design-documents/graph-ql/coverage/CartAddressOperations.graphqls]: I'm not sure if `customer_notes` field makes sense for a billing address.

File: design-documents/graph-ql/coverage/CartAddressOperations.graphqls [hunk 2]
```diff
@@ -96,15 +105,24 @@ type CartAddressRegion {
...
-    free_shipping: Boolean!
+type AvailableShippingMethod {
+    carrier_code: String!
+    carrier_title: String!
+    method_code: String
+    method_title: String
     error_message: String
-    # TODO: Add more complex structure for shipping rates
+    amount: Money!
+    base_amount: Money
```
Comment (joni-jones) [inline design-documents/graph-ql/coverage/CartAddressOperations.graphqls]: Why `base_amount` is exposed via GraphQL?

File: design-documents/graph-ql/coverage/CartAddressOperations.graphqls (hunk 1 above)
Comment (swnsma) [reply design-documents/graph-ql/coverage/CartAddressOperations.graphqls]: Synchronised with latest version of the schema.
The `customer_notes` is deprecated for Billing Address and removed in current Schema.

File: design-documents/graph-ql/coverage/CartAddressOperations.graphqls (hunk 2 above)
Comment (swnsma) [reply design-documents/graph-ql/coverage/CartAddressOperations.graphqls]: Synchronised with latest version of the schema.
The `base_amoun` is deprecated field and I have removed it from the schema.

**Pull request merged**